src/ssl.c

```diff
@@ -1205,6 +1205,33 @@ int wolfSSL_set_ConnectFilter(
 #endif /* WOLFSSL_WOLFSENTRY_HOOKS */
 
 #ifndef WOLFSSL_LEANPSK
+#ifdef WOLFSSL_WRAPPER_ALLOCATORS
+void* wolfSSL_dtls_create_peer(int port, char* ip)
+{
+    struct sockaddr_in *addr;
```

Comment:
Should you be using `SOCKADDR_IN` for portability?

Reply:
Yes i should. Fixed.